api/container.py

```diff
@@ -0,0 +1,21 @@
+class Container(object):
+    """
+        Defines the container aka fabric attributes
+    """
+    def __init__(self, fabric_height, fabric_max_length=None):
+        """
+            fabric_height: fixed input from user
```

Comment:
what we actually mean is the fabric width (fixed). Variable is fabric length and units are generally in yards